Userland/Applications/ThemeEditor/MainWidget.cpp

```diff
@@ -208,7 +208,9 @@ ErrorOr<void> MainWidget::initialize_menubar(GUI::Window& window)
         auto response = FileSystemAccessClient::Client::the().try_open_file(&window, "Select theme file", "/res/themes"sv);
         if (response.is_error())
             return;
-        load_from_file(*response.value());
+        auto load_from_file_result = load_from_file(*response.value());
+        if (load_from_file_result.is_error())
+            return;
```

Comment:
The general pattern for this in GUI apps is to throw up an "Error" message box saying like "Can't open file named {}: {}", filename, error

Reply:
I agree it would be better to handle it, I was erring on the side of caution here and repeating the same as the case just above. I think we should handle both and I'll update the commit to suit.